python/ray/util/sgd/torch/torch_trainer.py

```diff
@@ -170,13 +170,7 @@ def data_creator(config):
 
     def __init__(
             self,
-            *,
-            model_creator,
-            data_creator,
-            optimizer_creator,
-            loss_creator=None,
-            scheduler_creator=None,
-            training_operator_cls=None,
```

Comment:
be sure to leave these in there, but mark it as deprecated. This allows users to migrate easily.